plv8.h

```diff
@@ -48,7 +48,10 @@ class js_error
 {
 private:
 	char	   *m_msg;
+	int	   m_code;
```

Comment:
I'll do a close review before release - hence likely a release this weekend.  scary thought, but I actually have a pretty good handle on this stuff, just sometimes a bit slow :)